lib/index.js

```diff
@@ -71,21 +88,25 @@ function StickyInlines(opts) {
    * @return {Change}
    */
 
-  function onChange(change) {
-    if (!canBeEmpty) return null
-    const hasSave = change.flags.hasOwnProperty('save')
-    const toRemove = change.value.document.getInlines().reduce((failures, inline) => {
-      const hasFocus = change.value.isFocused && change.value.selection.hasEdgeIn(inline)
-      const onlyHasZeroWidthSpace = inline.text === ZERO_WIDTH_SPACE
-
-      if (isInlineBanned(inline, opts)) return failures
-      return !hasFocus && onlyHasZeroWidthSpace ? [inline, ...failures] : failures
-    }, [])
-
-    if (!toRemove.length) return null
-
-    toRemove.forEach((failure) => change = change.removeNodeByKey(failure.key))
-    return true
+  function onChange(change, next) {
+    if (!canBeEmpty) return next();
```

Comment:
Sorry, I realize this is old code but this is a little confusing to me and is smells like a performance issue. Is this going over the entire document to look for void inlines and rip them out? What's the purpose of this event handler? 

Might be a better question for @YurkaninRyan

Reply:
Hey Cameron, it's been awhile but we use zero-width spaces to keep "empty" inlines around.  This cleans them up if you unfocus them.

There may be a better way to do this.